Makefile

```diff
@@ -86,51 +90,82 @@ integration-test-full: integration-env-down integration-env-up integration-test
 
 integration-env-up-ci: emily-cdk-synth
 	docker compose --file docker/docker-compose.ci.yml up --detach --quiet-pull
-	@echo "Wait for aws resources to be set up..."
-	@while docker compose --file docker/docker-compose.ci.yml ps | grep -q 'emily-aws-setup'; do echo "waiting..." && sleep 1; done
+	DYNAMODB_ENDPOINT=http://localhost:8000 \
+		INPUT_CDK_TEMPLATE=./emily/cdk/cdk.out/EmilyStack.template.json \
+		OUTPUT_CDK_TEMPLATE=./emily/cdk/cdk.out/EmilyStack.devenv.template.json \
+		LOCAL_LAMBDA_PATH=empty.zip \
+		TRUSTED_REORG_API_KEY=testApiKey \
+		python3 docker/sbtc/emily-aws-setup/initialize.py
+	cargo build --bin emily-server
 	AWS_ACCESS_KEY_ID=xxxxxxxxxxxx \
-	AWS_SECRET_ACCESS_KEY=xxxxxxxxxxxx \
-	AWS_REGION=us-west-2 \
-	TRUSTED_REORG_API_KEY=testApiKey \
-	cargo run --bin emily-server -- \
-		--host 127.0.0.1 --port 3031 --dynamodb-endpoint http://localhost:8000 > ./target/emily-server.log 2>&1 &
+		AWS_SECRET_ACCESS_KEY=xxxxxxxxxxxx \
+		AWS_REGION=us-west-2 \
+		TRUSTED_REORG_API_KEY=testApiKey \
+		cargo run --bin emily-server -- \
+			--host 127.0.0.1 --port 3031 --dynamodb-endpoint http://localhost:8000 > ./target/emily-server.log 2>&1 &
```

Comment:
not strictly related to this but still: why can't we use `integration-env-up` directly even in CI?

Reply:
i think we could, but it would be slower

Reply:
Makes sense, it's just a bit annoying that we run something that is slightly different that the setup we use to run those locally.

Reply:
Yeah -- I had actually toyed with the idea of doing the reverse, i.e. using the ci version locally; I might still try to do this, but in another PR..